stories/plot-object.stories.tsx

```diff
@@ -11,6 +11,11 @@ export default {
 
 export function Control() {
   const plot: PlotObjectType = {
+    viewportStyle: {
+      stroke: 'black',
+      strokeWidth: '2px',
+      fill: '#111',
```

Comment:
does the `fill` property have any effect on the viewport given that `fillOpacity` is 0

Reply:
You are right, is needed to add `fillOpacity: 1` to change the color